include/highfive/bits/H5Node_traits_misc.hpp

```diff
@@ -175,6 +175,11 @@ inline std::vector<std::string> NodeTraits<Derivate>::listObjectNames() const {
 
 template <typename Derivate>
 inline bool NodeTraits<Derivate>::_exist(const std::string& node_name) const {
+    // The root path always exists, but H5Lexists return 0 or 1
```

Comment:
We can't assume H5Lexists("/") will succeed because exceptions can happen.
It's easier if we put this `if` in the routine below (below the call to this `_exist()`, so that eventual exceptions are correctly raised.